plugin/vscode/.vscodeignore

```diff
@@ -3,9 +3,6 @@
 !LICENSE
 !README.md
 !extension.js
-!node_modules/
-!node_modules/quick-lint-js-wasm/
-!node_modules/quick-lint-js-wasm/dist/
```

Comment:
Minor: I prefer what we have now, which works with as many versions of vsce as possible.